Felienne/hedy
# [BUG FIX] Fixes syntax highlighting on explore page

**Description**
This is a very very small bug fix, but important still. Somewhere we changed from using the `id` attribute as indicator for the ace editor level to using a dedicated `level` attribute. We forgot to align this change with the explore page where still the `id` attribute is set to the program level. In this PR we fix this by replacing `id` with `level`. 

**Fixes**
No related issue.

**How to test**
Login, navigate to the explore page and verify that the syntax highlighting is working.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
